doc/man1/openssl-rand.pod.in

```diff
@@ -14,12 +14,14 @@ B<openssl rand>
 [B<-hex>]
 {- $OpenSSL::safe::opt_engine_synopsis -}{- $OpenSSL::safe::opt_r_synopsis -}
 {- $OpenSSL::safe::opt_provider_synopsis -}
-I<num>
+I<num>[k|m|g|t]
 
 =head1 DESCRIPTION
 
 This command generates I<num> random bytes using a cryptographically
-secure pseudo random number generator (CSPRNG).
+secure pseudo random number generator (CSPRNG). the suffix k, m, g or t may be
+added to the num value to indicate the request value be scaled as a multiple of
+kiB/MiB/GiB/TiB respectively. Note that suffixes are case insensitive.
```

Comment:
It's `KiB` not `kiB`. (Yes, for decimal prefixes it's officially `kB` but for binary prefixes it's officially `KiB`. What! I have no idea why, maybe they just thought `kiB` looked too weird or something &mdash; I'd agree with that.)

Reply:
+1 - it's indeed KiB, see https://en.wikipedia.org/wiki/Byte

I think we should also add a note about the maximum amount of Bytes which can be requested.  SP 800-90C limits DRBG output to 2⁶⁴ **bits** per instantiation.

Reply:
I even stated it [copy&pasteable](https://github.com/openssl/openssl/pull/22624#issuecomment-1798285467) 🤷‍♂️

Reply:
> I even stated it [copy&pasteable](https://github.com/openssl/openssl/pull/22624#issuecomment-1798285467) 🤷‍♂️

...with two typos 🤣   (now fixed)

Reply:
Fixed kiB, added note about SP 800-90C limitation on rng generation